docs/bookmarks.md

```diff
@@ -0,0 +1,221 @@
+# Bookmarks
+
+
+## Introduction
+
+Bookmarks are named pointers to revisions (just like branches are in Git). You 
+can move them without affecting the target revision's identity. Bookmarks
+automatically move when revisions are rewritten (e.g. by `jj rebase`). You can
+pass a bookmark's name to commands that want a revision as argument. For example,
+`jj new main` will create a new revision on top of the "main" bookmark. Use
+`jj bookmark list` to list bookmarks and `jj bookmark` to create, move, or delete
+bookmarks. There is currently no concept of an active/current/checked-out bookmark.
+
+Currently Jujutsu maps its Bookmarks to Git Branches and stores them as that 
+in the Git backend. This means that all Bookmarks will be reflected as 
+Git Branches, this may change in the future. 
+
+## Remotes and tracked bookmarks
+
+Jujutsu records the last seen position of a bookmark on each remote (just like
+Git's remote-tracking branches). This record is updated on every `jj git fetch`
+and `jj git push` of the bookmark. You can refer to the remembered remote bookmark
+positions with `<bookmark name>@<remote name>`, such as `jj new main@origin`. `jj`
+does not provide a way to manually edit these recorded positions.
+
+A remote bookmark can be associated with a local bookmark of the same name. This is
+called a **tracked remote bookmark**. When you pull a tracked bookmark from a
```

Comment:
With Git remotes, at least, the local bookmark tracks a branch on the remote.

I'm not sure whether we should just explain that, or also rename "tracked remote bookmarks" to "tracked remote branches".

Reply:
I think it's fine here to explain that "tracked remote bookmarks" currently map to Git remote branches, and avoid the rename to keep it as _our_ concept.

Reply:
I agree that this is a reasonable option. We could put it into this PR or make a separate PR for it.

Reply:
I've added the sentence above here, is that ok for you?

Reply:
I think it helps quite a bit, but I have some nits. Let's continue this conversation in https://github.com/martinvonz/jj/pull/4341#discussion_r1747911931.